integ-test/build.gradle

```diff
@@ -113,27 +117,37 @@ testClusters.integTest {
     plugin ":opensearch-sql-plugin"
     keystore 'plugins.query.federation.datasources.config', new File("$projectDir/src/test/resources/catalog/", 'catalog.json')
 }
+
 task startPrometheus(type: SpawnProcessTask) {
     mustRunAfter ':doctest:doctest'
+
     doFirst {
         download.run {
-            src 'https://github.com/prometheus/prometheus/releases/download/v2.39.1/prometheus-2.39.1.linux-amd64.tar.gz'
+            src getPrometheusBinaryLocation()
             dest new File("$projectDir/bin", 'prometheus.tar.gz')
         }
         copy {
             from tarTree("$projectDir/bin/prometheus.tar.gz")
             into "$projectDir/bin"
         }
-        copy {
-            from "$projectDir/bin/prometheus.yml"
-            into "$projectDir/bin/prometheus-2.39.1.linux-amd64/prometheus"
+        file("$projectDir/bin").eachDir {
+            if (it.name.startsWith("prometheus-")) {
+                println "Renaming folder : " + it.name.toString()
+                println it.renameTo("$projectDir/bin/prometheus")
+            }
         }
     }
-    command "$projectDir/bin/prometheus-2.39.1.linux-amd64/prometheus --storage.tsdb.path=$projectDir/bin/prometheus-2.39.1.linux-amd64/data --config.file=$projectDir/bin/prometheus-2.39.1.linux-amd64/prometheus.yml"
+    command "$projectDir/bin/prometheus/prometheus --storage.tsdb.path=$projectDir/bin/prometheus/data --config.file=$projectDir/bin/prometheus/prometheus.yml"
     ready 'TSDB started'
 }
 
-task stopPrometheus(type: KillProcessTask)
+task stopPrometheus(type: KillProcessTask) {
+    doLast {
+        println "Deleting prometheus folder"
+        println file("$projectDir/bin/prometheus").deleteDir()
+        println file("$projectDir/bin/prometheus.tar.gz").delete()
+    }
```

Comment:
for debug?

Reply:
I am deleting the downloaded artifacts after the integ tests.

Reply:
Will remove println statements.